frontend/server.js

```diff
@@ -0,0 +1,18 @@
+function getArg(p) {
```

Comment:
I'm not familiar enough with JavaScript to say definitively what it is called but I would be confident there is either a standard argparser or a widely-adopted library that could be used for this.  This solution still serves to enable the friendlier local test environment which I will find very helpful.  It might become even more helpful for us to enable all kinds of command-line options as we progress, the library would make this quick and painless to extend to more command line options.

Reply:
There is stuff like URI.js but our needs are very simple, at least for now, so I'd rather avoid a dependency and associated licensing.